x-pack/plugins/observability_onboarding/public/routes/templates/custom_logs.tsx

```diff
@@ -57,6 +61,7 @@ function AnimatedTransitionsWizard() {
     <WizardProvider
       transitionDuration={TRANSITION_DURATION}
       onChangeStep={onChangeStep}
+      basePath="/customLogs"
```

Comment:
if routes can be generated, then basePath would be passed into `createWizardContext` and may not be necessary to pass in as a prop to the provider.

Reply:
Addressed in [27f765c](https://github.com/elastic/kibana/pull/161230/commits/27f765c86c5254d17f8804d6fb6fb679afd6d00f).